clientv3/op.go

```diff
@@ -263,6 +280,12 @@ func WithCountOnly() OpOption {
 	return func(op *Op) { op.countOnly = true }
 }
 
+// WithMinModRev filters out keys with modification revisions less than the given revision.
```

Comment:
add a comment about this option only works for range request? users might think this would work for deleterange.